src/components/TiktokScroll/index.js

```diff
@@ -22,6 +22,14 @@ class TiktokScroll extends React.Component {
     });
   }
 
+  componentDidMount() {
+    if (this.props.contentInsetAdjustmentBehavior === 'never') {
+      setTimeout(() => {
+        this.scrollToTop();
+      }, 200);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `componentDidMount` lifecycle method has been added to handle a specific case where `contentInsetAdjustmentBehavior` prop is set to 'never'. This will delay the execution of `scrollToTop` function by 200ms. However, it's important to note that using `setTimeout` in React components can lead to potential issues if the component unmounts before the timeout function executes. To avoid possible memory leaks, you should clear the timeout when the component unmounts.

```diff
+  componentDidMount() {
+    if (this.props.contentInsetAdjustmentBehavior === 'never') {
+      this.timeoutId = setTimeout(() => {
+        this.scrollToTop();
+      }, 200);
+    }
+  }
+
+  componentWillUnmount() {
+    if (this.timeoutId) {
+      clearTimeout(this.timeoutId);
+    }
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->